src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -156,7 +156,7 @@ function parseSwap(changes: TransactionChanges) {
       .join()
 
     return { title, descriptor }
-  } else if (changes.TokenTransfer.length === 2) {
+  } else if (changes.TokenTransfer.length >= 2) {
```

Comment:
FOT tokens have more than 2 transfers

Reply:
i would say leave this as a comment in the code too :D